app/controllers/idv/personal_key_controller.rb

```diff
@@ -65,11 +65,7 @@ def finish_idv_session
 
       irs_attempts_api_tracker.idv_personal_key_generated
 
-      if idv_session.address_verification_mechanism == 'gpo'
-        if !IdentityConfig.store.gpo_personal_key_after_otp
-          flash.now[:success] = t('idv.messages.mail_sent')
-        end
-      else
+      unless idv_session.address_verification_mechanism == 'gpo'
```

Comment:
As a personal style thing, I try to avoid `unless`:
```suggestion
      if idv_session.address_verification_mechanism != 'gpo'
```